lib/modules/manager/github-actions/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface Container {
+  image: string;
+}
+
+export interface Job {
+  container?: string | Container;
+  services?: Map<string, string | Container>;
+}
+
+export interface Workflow {
```

Comment:
Please  add some tsdoc and link to documentation

Reply:
First time using tsdoc, let me know if it is proper